Examples/MyExample(FactoryMethod)/ProgramExample.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Examples.MyExample_FactoryMethod_;
+
+public class ProgramExample
+{
+    public static void RunApplication()
+    {
+
+        ITransportFactory airFactory = new AirTransportationServiceFactory();
```

Comment:
it is not a good idea to connect your user with your factory in straightforward.
for this case it is enough.

you can follow the diagram of factory method for more information.